package.json

```diff
@@ -22,17 +22,16 @@
     "doc": "esy dune build @doc"
   },
   "dependencies": {
-    "@opam/dune": "*",
+    "@opam/dune": ">=1.11",
```

Comment:
Since you set `(lang dune 2.6)`, to avoid any possible issues I would say use version `>= 2.6` here